features/commit.feature

```diff
@@ -2,6 +2,14 @@ Feature: Creating a git commit from a Pivotal Tracker story
   Background:
     Given a file named "tmp/global_settings.yml" with "pivotal_tracker_api_key: my_api_key"
 
+
+  Scenario: If there are no stories, the command fails
+    Given there are no stories
+
+    When I run `geordi commit` interactively
```

Comment:
You only need "interactively" when you intend to interact with the CLI. In this case you should omit it in order to not confuse your fellow developers.